src/Components/Pagination/Pagination.js

```diff
@@ -0,0 +1,36 @@
+import { useParams } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const Pagination = (props) => {
+  const { total_pages, currentPage } = props;
+  let { q } = useParams();
```

Comment:
`let` and `const` are meant to be used differently, `let` is used when a variable is going to be reassigned, for example:

```js
let value = { a: 1 };
...
value = { b: 2 }
```

and `const` is meant to be used when the value won't change, when working with objects/functions it is preferable to create a new variable than mutate an existing one, dealing with inmutable variables makes the code easier to maintain